src/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const initialState = {
+  data: [],
+  status: 'idle',
+  error: null,
+};
+
+export const fetchRocketsData = createAsyncThunk('rockets/fetchData', async () => {
+  const response = await axios.get('https://api.spacexdata.com/v3/rockets');
+  const rocketsData = response.data.map((rocket) => ({
+    id: rocket.id,
+    name: rocket.name,
+    type: rocket.type,
+    flickr_images: rocket.flickr_images,
+  }));
+  return response.data;
+});
```

Comment:
Hello @DianaBeki,

I feel you misunderstood my suggestion here. 

The fetchRocketsData function was good as it was, you just needed to add data filtration to the fetched data. I noticed you took off the try and catch error from it while implementing my suggested changes. 

Please replace your fetchRocketsData with the following code.

```suggestion
 export const fetchRocketsData = createAsyncThunk('rockets/fetchData', async () => {
  try {
    const response = await axios.get('https://api.spacexdata.com/v3/rockets');
    const rocketsData = response.data.map((rocket) => ({
      id: rocket.id,
      name: rocket.name,
      type: rocket.type,
      flickr_images: rocket.flickr_images,
    }));
    return rocketsData;
  } catch (error) {
    throw new Error('Unable to fetch data for the rockets');
  }
});
```